app.py

```diff
@@ -9,6 +9,7 @@
 import requests
 app = Flask(__name__)
 app.config.from_object(Config)
+app.logger.setLevel(app.config['LOG_LEVEL'])
```

Comment:
Doesn't this conflict with the instruction to add 
```
import logging
logging.basicConfig(level=logging.INFO)
```
near the start of the exercise?